flutter-gui-tests/testSrc/io/flutter/tests/gui/SmokeTest.kt

```diff
@@ -0,0 +1,62 @@
+package io.flutter.tests.gui
+
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.ui.MultiLineLabelUI
+import com.intellij.openapi.util.text.StringUtil
+import com.intellij.testGuiFramework.framework.RunWithIde
+import com.intellij.testGuiFramework.impl.GuiTestCase
+import com.intellij.testGuiFramework.launcher.ide.CommunityIde
+import org.junit.Assert
+import org.junit.Test
+
+// Inspired by CommandLineProjectGuiTest
+@RunWithIde(CommunityIde::class)
+class SmokeTest : GuiTestCase() {
+  private val LOG = Logger.getInstance(this.javaClass)
+
+  @Test
+  fun testStartup() {
+    ProjectCreator.createProject(projectName = "guitest")
+    ideFrame {
+      editor {
+        // Wait until current file has appeared in current editor and set focus to editor.
+        moveTo(1)
+      }
+      val editorCode = editor.getCurrentFileContents(false)
+      Assert.assertTrue(editorCode!!.unifyCode().startsWith(codeText.unifyCode()))
```

Comment:
I guess the templates don't change often but my inclination would be to make this a little fuzzier in case they do.

(On the other hand, we can just cross this bridge if/when we need to.)

Reply:
Yeah, I could probably just check that the content is not empty. We don't really care what it is, do we?

Reply:
Right.  You'd hope this is being well tested elsewhere.